src/router.js

```diff
@@ -0,0 +1,28 @@
+export default class Router {
+  constructor(routes, element) {
+    this.routes = routes;
+    this.rootElement = element;
+    this.init();
+  }
+
+  init() {
+    window.addEventListener('hashchange', (url) => { this.render(url); });
+    window.location.hash = 'login';
```

Comment:
window.location.hash = 'login';
This thing will be moved to guards for a routes file